torch/csrc/api/src/nn/module.cpp

```diff
@@ -292,14 +292,9 @@ void Module::load(serialize::InputArchive& archive) {
     archive.read(buffer.key(), buffer.value(), /*is_buffer=*/true);
   }
   for (const auto& child : children_) {
-    // Modules that have no state at all (parameters or buffers) are currently
-    // not stored in Protobuf at all, so we can just skip them.
-    if (!child.value()->parameters_.is_empty() ||
-        !child.value()->buffers_.is_empty()) {
-      serialize::InputArchive child_archive;
-      archive.read(child.key(), child_archive);
-      child.value()->load(child_archive);
-    }
+    serialize::InputArchive child_archive;
+    archive.read(child.key(), child_archive);
+    child.value()->load(child_archive);
```

Comment:
Uh... is this the only actual substantive change?

Reply:
Yes, this is the bug